apps/website/src/pages/docs/avatar.mdx

```diff
@@ -17,81 +17,59 @@ import * as AllExamples from 'examples';
   <AllExamples.AvatarMainExample client:load />
 </LiveExample>
 
-The avatar is an essential UI component, making it easier to identify a person or organization by their photo, icon, or initials. A user’s photo helps put a face to the name and makes it more personal. Any time an app requires a user profile, an avatar should be included. It also makes it very obvious to the user that they are logged in, as the non-logged in state displays differently.
+The avatar is an essential UI component, making it easier to identify a person or organization by their photo, icon, or initials.
 
-## Appearance
+## Usage
 
-### Sizes
-
-There are four sizes of avatars:
-
-<LiveExample src='Avatar.sizes.tsx'>
-  <AllExamples.AvatarSizesExample client:load />
-</LiveExample>
-
-- `small`
-- `medium`
-- `large`
-- `x-large`
-
-Which size to use is at your discretion. Determine which size is best for your context based on available space.
-
-### Initials
+Different properties can be passed to the `<Avatar>` component to customize it according to the user's needs.
 
-If a user does not upload a photo, the user’s first & last initials are displayed. You can display a maximum of 2 letters and a minimum of 1 letter. In the off chance there are user accounts without any name associated, you can display the first letter of the user’s email address.
+- `abbreviation` prop allows users to provide a short abbreviation or initials for the avatar.
+- `image` prop can be passed to set a custom image source or icon that will be displayed within the avatar.
```

Comment:
1. `image` should be moved up because it's the preferred way to use Avatar.
2. `abbreviation` should mention that it will be displayed as fallback without an image

```suggestion
- `image` prop can be passed to set a custom image source or icon that will be displayed within the avatar.
- `abbreviation` prop can be passed to set a short abbreviation or initials that will be displayed when `image` is not available.
```